src/js/src/components/SearchSingleItemComponents/SearchSingleItemAllData.vue

```diff
@@ -5,51 +5,121 @@
         {{ allDataButtonText }}
       </button>
     </div>
-    <div v-if="allDataShown">
+    <div v-if="allDataShown && allData !== {}">
       <hr class="informationDivider">
       <div class="table">
         <div class="tr">
           <span class="td highlightText">Attribute</span><span class="td highlightText">Value</span>
         </div>
-        <div v-for="(key, index) in Object.entries(result)" :key="index" class="tr">
-          <span class="td">{{ key[0] }}</span> <span class="td">{{ key[1] }}</span>
+        <div v-for="(key, index) in Object.entries(allData)"
```

Comment:
Find a better word than 'key' for describing the data item you are displaying - this will help the next coder in reading the complex logic below.

Reply:
Yea - i did some renaming :D

Reply:
👍